### PR TITLE
RESOURCE-152 Support azure power bi app reports

### LINKED ISSUE #462: Power BI App Reports

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Support azure power bi app report(s)

### Issues Resolved
#462 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/azure_power_bi_app_reports.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  describe azure_power_bi_app_report(app_id: 'APP_ID', report_id: id) do
+    it { should exist }
+  end
+end
+```
+
+### Test to filter out Power BI App Reports that have webUrl
+
+```ruby
+describe azure_power_bi_app_reports(app_id: 'APP_ID').where(name: 'SalesMarketing') do
```
Comment (sa-progress) [inline docs/resources/azure_power_bi_app_reports.md:72]: Should we use the same convention we have started using like .where(name: 'NAME')?

**Review by sa-progress — APPROVED** [review]
Please check the comment .Apart from that everything looks good.

File: docs/resources/azure_power_bi_app_report.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/power-bi/apps/get-report) for other properties available.
+
+## Examples
+
+### Test that the Power BI App report is at the left corner.
+
+```ruby
+describe azure_power_bi_app_report(app_id: 'APP_ID', report_id: 'REPORT_ID')  do
+  its('rowSpan') { should eq 0 }
+  its('colSpan') { should eq 0 }
```
Comment (IanMadd) [inline docs/resources/azure_power_bi_app_report.md:65]: Is this correct?

File: docs-chef-io/content/inspec/resources/azure_power_bi_app_report.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+## Examples
+
+**Test that the Power BI App report is at the left corner.**
+
+```ruby
+describe azure_power_bi_app_report(app_id: 'APP_ID', report_id: 'REPORT_ID')  do
+  its('rowSpan') { should eq 0 }
+  its('colSpan') { should eq 0 }
+end
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/azure_power_bi_app_report.md:75]: Is this correct?

**Review by IanMadd — APPROVED** [review]

**Pull request merged**